### ISSUE TITLE
Support for other languages

### ISSUE THREAD

**Issue by pklaschka** [talal/ilm#2]
First of all: thanks for this fantastic template! I've already used it in a few documents. One feature request I have, though, would be to support other languages apart from English (currently, things like the date format and headings "force" English in some areas). From a brief look at the template, it looks like this should be relatively easy to achieve, apart from one area:

## Trivial Cases

Here, hard-coded values could simply become parameters with default values:

https://github.com/talal/ilm/blob/ea4413dbb2c43b2b715b5879d8d87432c5904432/lib.typ#L93-L94

https://github.com/talal/ilm/blob/ea4413dbb2c43b2b715b5879d8d87432c5904432/lib.typ#L129-L130

https://github.com/talal/ilm/blob/ea4413dbb2c43b2b715b5879d8d87432c5904432/lib.typ#L236-L238

## Non-trivial Case

https://github.com/talal/ilm/blob/ea4413dbb2c43b2b715b5879d8d87432c5904432/lib.typ#L137-L139

Here, it isn't quite as easy, since the title of the bibliography currently isn't managed by the `lib.typ`. While, as a short-term solution, this variable could be introduced (and at least set as a default using a `#set` expression), I don't think this is a great long-term solution 🤔. A more stable solution could be to scope that chapter `pagebreak` show rule to only the `body`, with the bibliography and outlines being outside the affected scope.

## Next steps

I'd be willing to work on / create a PR for this. Would you accept such a change to the project? And if so: what's your preferred solution on the non-trivial case?

Thanks!

**Comment by talal** [comment]
Language support is a great idea. In hindsight, I should've made the different arguments for the template customizable to begin with 😅 

I've made the necessary changes to the template. All your concerns should be addressed in v1.0.0. Here's a summary of what you're probably looking for:

```
#show: ilm.with(
  ...

  # custom format string for date
  date-format: "",
  table-of-contents: outline(title: "Your custom title for TOC"),
  bibliography: bibliography("refs.bib", title: "Custom title"),
  figure-index: (enabled: true, title: "Custom title"),
  table-index: (enabled: true, title: "Custom title"),
  listing-index: (enabled: true, title: "Custom title")
)
```

You can wait until https://github.com/typst/packages/pull/520 gets merged or if you're using the template locally then just update the template to the latest version.

Let me know if there's anything still missing.

**Comment by pklaschka** [comment]
That was quick 👀!

Thank you so much, I'll give it a shot tomorrow!
